utils/string.ts

```diff
@@ -4,16 +4,66 @@ export function kebabCase(name: string) {
 	return name.replace(KebabCaseRegex, (letter, pos) => (pos ? '-' : '') + letter.toLowerCase());
 }
 
-export function titleCase(str: string) {
-	// Hyphen and underscore.
-	str = str.replace(/(\-|_)/g, ' ');
+export type TitleCaseOptions = Partial<{
```

Comment:
You're doing this as part of the title changes?

Reply:
yup